Makefile

```diff
@@ -14,13 +14,17 @@ endif
 COMPOSE_IGNORE_ORPHANS=1
 export
 
+export EVERY_LAMBDA_COMPOSE_FILE=-f docker-compose.lambda-zips.js.yml \
+	-f docker-compose.lambda-zips.python.yml \
+	-f docker-compose.lambda-zips.rust.yml
```

Comment:
Should we use `--file` here instead of `-f`? I see `EVERY_COMPOSE_FILE` doesn't, but I don't think that means we shouldn't use `--file` moving forward.

Reply:
Yeah, using `--file` would be the proper thing to do.